cScripts/Loadouts/CfgLoadouts_Charlie.hpp

```diff
@@ -2,7 +2,7 @@
 
 class CAV_Charlie_OFFCR : CAV_Charlie_Base {  // Leadership positions PSG and up
 
-    backpack[] = {"B_Carryall_mcamo"};
+    backpack[] = {"B_AssaultPack_mcamo"};
```

Comment:
Can be removed due to being defined in Charlie base as of #267

Reply:
updated